content-src/components/TopSites/TopSite.jsx

```diff
@@ -281,11 +281,7 @@ export class TopSite extends React.PureComponent {
     const title = link.label || link.hostname;
     return (<TopSiteLink {...props} onClick={this.onLinkClick} onDragEvent={this.props.onDragEvent} className={`${props.className || ""}${isContextMenuOpen ? " active" : ""}`} title={title}>
         <div>
-          <button className="context-menu-button icon" title={this.props.intl.formatMessage({id: "context_menu_title"})} onClick={this.onMenuButtonClick}>
-            <span className="sr-only">
-              <FormattedMessage id="context_menu_button_sr" values={{title}} />
-            </span>
-          </button>
+          <button aria-haspopup="true" className="context-menu-button icon" aria-label={this.props.intl.formatMessage({id: "context_menu_title"})} title={this.props.intl.formatMessage({id: "context_menu_title"})} onClick={this.onMenuButtonClick} />
```

Comment:
This removes the `title` from the screenreader -- intended?

Reply:
My understanding is that the screen reader will pick up either `title` or `aria-label` as long as they're both present, so as long as we're seeing one or the other it should be fine

Reply:
Sorry. `title` was ambiguous. I was referring to the js value `values={{title}}` not the html attribute.

Reply:
ah okay. Is `values={{title}}` able to be interpreted by anything besides the `<FormattedMessage>` component? @k88hudson and I had gone through and taken them out.

Reply:
Should be able to pass in a values object https://github.com/formatjs/react-intl/blob/master/docs/API.md#formatmessage

Reply:
ahh that makes sense!